mapping/README.md

```diff
@@ -147,19 +148,53 @@ If transform is not specified values will be transformed as is.
 
 A Math transformation can be defined by the `math` attribute.
 It accepts [py-expression-eval](https://github.com/AxiaCore/py-expression-eval/) formulas as argument.
-The DBC feeder expects the DBC value to be represented as `x` like in the example below.
+The DBC feeder expects the DBC value to be represented as `x`.
+
+When evaluating what transformation that is needed one must study both the DBC signal and the VSS signal. An example is given below for mirror tilt of left mirror.
+
+**DBC**
+
+The signal `VCLEFT_mirrorTiltYPosition` provide mirror tilt.
+
+```
+SG_ VCLEFT_mirrorTiltYPosition : 41|8@1+ (0.02,0) [0|5] "V"  Receiver
+```
+
+An introduction to DBC file syntax can be found [here](https://www.csselectronics.com/pages/can-dbc-file-database-intro). The specification above shows that on CAN the tilt is represented as 0-5 Volts. The value is sent with a scaling of 0.02 and uses 8 bits, i.e. 5 Volts is transmitted as 250, but that information is not needed for the can-provider transformation as the DBC scaling/offset is performed automatically by the CAN middleware.
```

Comment:
I would say

"... is not needed when configuring the mapping of the kuksa-can-provider, as the transformation defined in the DBC is performed automatically when CAN frames are read. For the input to the mapping you will need to consider just a value between 0 and 5 V in this example.